oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -35,11 +35,11 @@ Context
 
 Developers who work on the Open edX platform, both inside and outside of edX, need accurate and current documentation about platform architecture, APIs, and development best practices.
 
-**Developer documentation** is defined as written specification, instructions, and resources for software developers who extend and contribute to the Open edX platform. It does not include documentation resources targeted for edX learners, educators, researchers, and Open edX operators. Those are out of scope of this OEP and maintained by the edX documentation team at docs.edx.org_.
+**Developer documentation** is defined as written specification, instructions, and resources for software developers who extend and contribute to the Open edX platform. It does not include documentation resources targeted for edX learners, educators, researchers, and Open edX operators. Those are out of scope of this OEP and maintained by the edX at docs.edx.org_.
 
 Developer documentation is inconsistent and poorly maintained. Documents are also difficult to find. They exist in multiple locations, including `Read the Docs`_, GitHub_ repositories, Confluence_ wiki, and `Google Docs`_.
 
-This proposal focuses on long-term developer *Documentation* needs. It does not concern itself with any transitory *Notes* that a development team may use for project management, brainstorming, meeting notes, etc. Code *Comments* that a developer may add to clarify or rationalize a code segment are also out of scope of this OEP.
+.. This proposal focuses on long-term developer *Documentation* needs. It does not concern itself with any transitory *Notes* that a development team may use for project management, brainstorming, meeting notes, etc. Code *Comments* that a developer may add to clarify or rationalize a code segment are also out of scope of this OEP.
```

Comment:
Doesn't this just make it a comment? Can you explain this change? Is this already noted elsewhere? I think we should minimize unpublished notes, and either delete this or make it more brief, or move it, if you think it isn't relevant.